packages/sources/coingecko/src/endpoint/crypto.ts

```diff
@@ -1,4 +1,5 @@
-import { Requester, Validator, Logger } from '@chainlink/ea-bootstrap'
+import { Requester, Validator, Overrider, Logger } from '@chainlink/ea-bootstrap'
+import { AdapterError, OverrideToOriginalSymbol } from '@chainlink/ea-bootstrap/src/lib/modules'
```

Comment:
AdapterError is a root level export from `ea-bootstrap`. `OverrideToOriginalSymbol` should come from `Overrider.OverrideToOriginalSymbol` or from the repo level types.